skyportal/handlers/api/source.py

```diff
@@ -491,6 +492,24 @@ def get(self, obj_id=None):
             description: |
               Comma-separated string of "taxonomy: classification" pair(s) to filter for sources matching
               that/those classification(s), i.e. "Sitewide Taxonomy: Type II, Sitewide Taxonomy: AGN"
+          - in: query
+            name: annotationsFilter
+            nullable: true
+            schema:
+              type: array
+              items:
+                type: string
+            explode: false
+            style: simple
+            description: |
+              Comma-separated string of "annotation: value: operator" triplet(s) to filter for sources matching
+              that/those annotation(s), i.e. "redshift: 0.5: lt"
+          - in: query
+            name: annotationsFilterOrigin
+            nullable: true
+            schema:
+              type: string
+            description: Comma-separated string of who posted the annotations to filter
```

Comment:
Origin is not exactly the same as who posted. I would change it to "comma separated string of origins. Only annotations from these origins are used when filtering with the annotationsFilter"

Reply:
Fixed.